Persistence/OwnerRepositoryDummy.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Persistence.Models;
+using RI.Novus.Core.Boundaries.Persistence;
+using RI.Novus.Core.Inmovable.Owners;
+
+namespace Persistence;
+
+/// <summary>
+/// Dummy implementation of <see cref="IOwnerRepositoryDummy"/> that uses an in-memory database.
+/// </summary>
+public sealed class OwnerRepositoryDummy: IOwnerRepositoryDummy
```

Comment:
```suggestion
public sealed class OwnerRepository: IOwnerRepository
```
This is not a dummy repository, because is connected to a database